### PR TITLE
(PA-2578) Cover tickets for puppet-runtime.

### PR DESCRIPTION
**Pull request by tavilarion**
This is required because a few tickets which are part of puppet-runtime were missing fix versions during the last release and were ignored by the script.

### CODE REVIEW

**Review by gimmyxd — APPROVED** [review]

File: pa_matchbatch.sh [hunk 1]
```diff
@@ -279,7 +315,13 @@ for currentItem in ${repoRevMap}; do
 		jiraProjectId=$(getJiraProjectIdFor "${public_name}")
 		jiraFixedInProject=$(getJiraFixedInFor "${public_name}")
 
-		versionsUsed="${versionsUsed}\n${public_name}:${fix_ver}"
+        if [[ $public_name = "puppet-runtime" ]]; then
```
Comment (speedofdark) [inline pa_matchbatch.sh]: this block needs to be indented to match the rest of the pushd/popd block.

File: pa_matchbatch.sh [hunk 2]
```diff
@@ -222,6 +226,27 @@ cloneOrFetch() {
 	fi
 }
 
+readRuntimeVersion() {
```
Comment (speedofdark) [inline pa_matchbatch.sh:229]: Would it make sense to fold this into getFixVerFor()? Or is the intent different?

Comment (tavilarion) [reply pa_matchbatch.sh:229]: The intent here is a bit different, the version is read from puppet-agent (puppet-runtime.json), the idea being to check all the changes between the version included in the last release (tag) and the current version.

File: pa_matchbatch.sh (hunk 1 above)
Comment (tavilarion) [reply pa_matchbatch.sh]: This is weird, it seems to be indented on my local copy, will fix this.

**Pull request merged**